src/uct/ib/ud/base/ud_ep.h

```diff
@@ -188,9 +188,9 @@ enum {
     UCT_UD_EP_FLAG_ASYNC_COMPS       = UCS_BIT(0), /* set if there are completions that
                                                     * were picked by async thread and queued */
     UCT_UD_EP_FLAG_DISCONNECTED      = UCS_BIT(1), /* set if the endpoint was disconnected */
+    UCT_UD_EP_FLAG_PRIVATE           = UCS_BIT(2), /* EP is was created as internal */
```

Comment:
where do you set it?

Reply:
https://github.com/openucx/ucx/blob/master/src/uct/ib/ud/base/ud_ep.c#L507